src/pages/console/data/data-model.mdx

```diff
@@ -29,43 +35,44 @@ Skip ahead to [relationships](/console/data/relationships) or read below on how
 
 ![datamodel](/images/console/datamodel.gif)
 
 The experience for modeling data is the same in the sandbox and in Studio. With an AWS account, you will have real-time data synchronization capabilities and you will also be able to set authorization rules on your models. For more information, see [Authorization](/console/authz/authorization). All data models are provisioned in your account as AWS AppSync GraphQL APIs and Amazon DynamoDB tables. As with every feature of Studio, it can be further extended with the CLI.
 
 ### DataStore and GraphQL
-By default, all data models built with Studio are provisioned with Amplify DataStore. However, Studio applications work with both DataStore and an Amplify GraphQL API. 
-
-**[DataStore](/lib/datastore/getting-started)** is an on-device storage engine that automatically synchronizes data between your mobile and web apps and your database, enabling offline use. DataStore requires a **Conflict Resolution** method to function. 
-
-An **[Amplify GraphQL API](../../cli/graphql/overview/)** is an online-only API that uses a GraphQL and DynamoDB via AWS AppSync. 
+When you save and deploy your data model, Amplify Studio generates a GraphQL API for you using AWS AppSync. By default, all data models built with Studio are provisioned as GraphQL APIs without DataStore - however, **Studio supports all Amplify GraphQL APIs, with and without DataStore**. 
 
-To change your app's Conflict Resolution settings, or to change your app from using DataStore to using an Amplify GraphQL API, take the following steps:
-1. Navigate to **[Amplify Studio](/console/adminui/start/#log-into-the-amplify-console) > Data**
-2. Select **GraphQL API settings** at the top of the page
-3. Toggle DataStore off or on, or change your Conflict Resolution strategy
-4. Return to the Data Modeling screen and select **Save and Deploy**
+**[DataStore](/lib/datastore/getting-started)** is an on-device storage engine that automatically synchronizes data between your mobile and web apps and your database, enabling offline use. DataStore is automatically added to your API when you add a **Conflict Resolution** method. 
 
 <Callout warning>
 
-**IMPORTANT NOTE**: Changing between DataStore and an Amplify GraphQL API is a significant change to your data structure, and can take up to 30 minutes to complete. Frequently changing between data structures is not recommended.
+**IMPORTANT NOTE**: Changing your Conflict Resolution setting results is a **destructive data change**, and may result in inaccessible data. It's highly recommended that this setting only be changed on GraphQL APIs with **no data**. 
```

Comment:
Really??? How so?

Reply:
Turning off Conflict Resolution deletes the columns (and the subsequent data) used in the conflict resolution strategy. These columns aren't needed if Conflict Resolution is disabled, so turning off Conflict Resolution is a **destructive** change, but a **non-breaking** data change. However, if the user then turns Conflict Resolution back on, that data will be inaccessible until they manually backfill the data in those re-added columns. 

This is a pretty nuanced difference that I didn't dive into in the docs because it's kinda complicated. I'm thinking maybe we should add a detailed paragraph explaining why the change is destructive below the callout.

Reply:
Confirmed that my info was wrong - updated this section accordingly